windows/src/engine/keyman32/Keyman32.cpp

```diff
@@ -427,6 +427,11 @@ extern "C" BOOL _declspec(dllexport) WINAPI Keyman_Initialise(HWND Handle, BOOL
 
 	InitDebugging();
 
+  if (!Globals::InitSettings()) {
```

Comment:
atm, isn't `Globals::InitSettings()` always `TRUE`?

Reply:
At present it is, yes. But this way if we add more stuff to `Globals::InitSettings` that could fail, we don't need to remember to go look at the caller to see how it handles failure. :)